### ISSUE TITLE
mothur won't run - system crash

### ISSUE THREAD

**Issue by stanleewild** [mothur/mothur#798]
I tried several times, but it always freezes my laptop. It was working well like 1 year ago, now when I want it start with icon or even in terminal as ./mothur, it always end up freezing my launcher and I have to do the hard reset. System is the newest Mac OS, 16GB RAM. Anybody had this issue?
Thanks!

**Comment by mothur-westcott** [comment]
I am happy to help you troubleshoot the issue.

What version of OS X are you running?

What version of mothur? Here's a link to our latest release, https://github.com/mothur/mothur/releases/tag/v1.46.1.

**Comment by stanleewild** [comment]
Hello Sarah

My OS is Big Sur 11.6 and the version of mothur is the most actual one -
1.46.1
I tried it in different ways but always I got that freeze where I had to do
a hard reset of my laptop.
Also, I tried to follow the manual but using "makefile" (step after
unziping) does not do anything for me ("make: *** No targets specified and
no makefile found.  Stop."). Is it important?
I wanted to use mothur for OTU analysis as I found it very useful in
combination with bGMYC for estimating the species delimination by changing
cutoff value and comparing number of OTUs with each other.
Thank you.


Stanislav Kolencik
